inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -164,6 +165,8 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     fq_fusions->add_matcher<ngraph::pass::ReluFakeQuantizeFusion>();
     fq_fusions->set_name("ngraph::pass::FakeQuantizeFusions");
 
+    manager.register_pass<ngraph::pass::StrideOptimization>();
```

Comment:
Yeah, maybe having StrideOptimization after FakeQuantizeFusion looks weird so I moved it before that. But I think it's beneficial for StrideOptimization to be placed near the end of CommonOptimization passes - so it can work on (in general) smaller function.

Reply:
During CommonOptimizations we do a lot of decompositions. So the graph will be bigger and may contain ops that we can not handle. Also we are trying to keep first part of common optimizations pipeline as plugin independent and second part as the part that plugins may parametrize.

Reply:
If we put StridesOptimization pass too early, we would end up with unnecessary MaxPools which may block other fusions.